package.json

```diff
@@ -96,6 +97,8 @@
     "sitemap": "node bin/build_sitemap.js",
     "lint": "eslint src --max-warnings 0",
     "pretest": "npm run lint",
+    "preserve": "npm run build",
+    "serve": "live-server --no-browser --entry-file=index.html --proxy=/api:$API_URL/api dist/",
```

Comment:
Can I confirm with you the `live-server` is only for development (ie. not a production server)? If so, when should we use this instead of the webpack-dev-server?

Reply:
Oh, never mind. This is for serving the built code in `dist`.

Reply:
Yeah it's to test a "production-ish" environment locally. It could probably use a better name.